packages/block-editor/src/components/border-radius-control/index.js

```diff
@@ -49,7 +50,9 @@ export default function BorderRadiusControl( { onChange, values } ) {
 		! hasDefinedValues( values ) || ! hasMixedValues( values )
 	);
 
-	const units = useCustomUnits( { availableUnits: [ 'px', 'em', 'rem' ] } );
+	const units = useCustomUnits( {
+		availableUnits: useSetting( 'spacing.units' ) || [ 'px', 'em', 'rem' ],
+	} );
 	const unit = getAllUnit( values );
 	const unitConfig = units.find( ( item ) => item.value === unit );
```

Comment:
When `availableUnits` is set empty, `useCustomUnits` returns `false`, which in turn throws an error in this line. Test steps:

- Set `settings.spacing.units` to an empty array.
- Select a group block and click the border panel.

Reply:
Thanks 👍 

This has been fixed. Testing with and without units now works as intended and free from errors.